python/its-interqueuemanager/its_iqm/authority/__init__.py

```diff
@@ -1,3 +1,25 @@
-import its_iqm.authority.file
-import its_iqm.authority.http
-import its_iqm.authority.mqtt
+from __future__ import annotations
+import its_iqm.authority.file as _file
+import its_iqm.authority.http as _http
+import its_iqm.authority.mqtt as _mqtt
+import its_iqm.iqm
+
+
+def Authority(
+    instance_id: str,
+    cfg: dict,
+    update_cb: Callable[[its_iqm.iqm.IQM, dict], None],
```

Comment:
> add `from typing import Callable`?

Not sure how this mess works, in fact, but since the file starts with:
```py
from __future__ import annotations
```
I believe the typing hints are then available...